### DISCUSSION TITLE
Google Storage Bucket - VM Dataset Connection on Training

### DISCUSSION THREAD

**Discussion by Spawnfile** [ultralytics/yolov5#9419] [Q&A] [answered]
Hi all,

I've got bunch of image dataset nearly 300 GB on GCS bucket which tracked by DVC. I want to train this images on VM instance without  transfering to VM instance local storage, but I saw a pro-tip that training mounted or network datasets would be very slow. How can I do that properly, any suggestions ?

**Comment by glenn-jocher** [comment, marked as answer]
@Spawnfile network drives are very slow, so you'll want to transfer the data once to your VM and then train locally from there.

On GCP you probably want to use a large SSD to provide sufficient IOPS and throughput, i.e. maybe 1TB.

**Reply by Spawnfile** [reply]
I've changed the pipeline to transfering data to VM avoid slow training caused by dataset. Thank you.
